src/devices_models/devices/common/get_time_series.jl

```diff
@@ -7,8 +7,11 @@ function get_time_series(
     @debug initial_time
     use_forecast_data = model_uses_forecasts(psi_container)
     time_steps = model_time_steps(psi_container)
-    @debug "device $(PSY.get_name(device)) $(PSY.has_forecasts(device)) forecast"
-    if use_forecast_data & PSY.has_forecasts(device)
+    has_forecasts = PSY.has_forecasts(device)
+    if !has_forecasts
+        @warn("device $(typeof(device)) $(PSY.get_name(device)) has forecasts = $(PSY.has_forecasts(device))")
+    end
```

Comment:
Perhaps this shouldn't be logged if the user has not requested forecasts to be used.
Also, here is a suggestion to simplify the log message.

```suggestion
    if use_forecast_data && !has_forecasts
        @warn "$(summary(device)) does not have forecasts)"
    end
```